### PR TITLE
Update composer.json

### PR DESCRIPTION
**Pull request by kunicmarko20**
| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | yes/no
| Deprecations? | yes/no
| Fixed tickets | #22
| License       | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,6 @@
         "symfony/routing": "^2.7 || ^3.0 || ^4.0"
     },
     "conflict": {
-        "egeloen/ckeditor-bundle": "*",
```
Comment (xabbuh) [inline composer.json]: Are you sure that works? Don't the class aliases conflict then?

Comment (kunicmarko20) [reply composer.json]: We went without class aliases, we migrated to FOS namespace.

Comment (xabbuh) [reply composer.json]: I see, should have checked that before

Comment (kunicmarko20) [reply composer.json]: https://github.com/FriendsOfSymfony/FOSCKEditorBundle/blob/master/doc/migration.rst here we wrote a guide on how to migrate.

**Pull request merged**